### DISCUSSION TITLE
RMA module

### DISCUSSION THREAD

**Discussion by techweb-digimedia** [breezefront/community#170] [3. Integrations]
Hello,

is there a module for RMA management, compatible with the breeze theme? Could this Amasty module (https://amasty.com/rma-for-magento-2.html) be compatible out of the box?
Thanks

**Comment by vovayatsyuk** [comment]
Looks like all complex functionality of this module is located under `/rma/` url. As a workaround, you can disable the breeze module at these pages using "Disable Breeze for specified URLs" [option](https://breezefront.com/docs/settings).

However, the [styles need to be integrated](https://breezefront.com/docs/integration-tips#migrate-module-css) even in this case.

**Comment by dimdimich** [comment]
One more advice also. Please check https://swissuplabs.com/blog/interested-in-amasty-modules-integration-with-breeze-check-this-information/.

We had discussion with Amasty and they told that if they will have enough request on their modules integration they will do that from their side.

So you can create separate Amasty RMA  integration requests there https://github.com/orgs/breezefront/discussions and then contact Amasty and let them know that yo uare using Breeze and interested in their module. In case it will be just one request then they will offer paid integration most likely but more people will ask more chances that they will add Breeze support from their side.

**Comment by techweb-digimedia** [comment]
Ok, thanks for the advices!
